js/utils.ts

```diff
@@ -1,4 +1,4 @@
-import { SidebarScrollState } from "./index";
+import { SidebarScrollState } from "./types";
```

Comment:
There were some circular imports going on, where `cljdoc.ts` was importing types from `index.tsx` and `index.tsx` was importing a bunch of functions from `cljdoc.ts`.